tf_quant_finance/black_scholes/approximations/american_option.py

```diff
@@ -343,6 +343,210 @@ def _calc_q(n, m, sign, k=1):
       (n - 1) ** 2 + tf.math.divide_no_nan(4 * m, k))) / 2
 
 
+def bjerksund_stensland_1993(volatilities,
+                 strikes,
+                 expiries,
+                 spots=None,
+                 forwards=None,
+                 discount_rates=None,
+                 dividend_rates=None,
+                 cost_of_carries=None,
+                 discount_factors=None,
+                 is_call_options=None,
+                 dtype=None,
+                 name=None):
+  """Computes the Bjerksund-Stensland 1993 price for a batch of call or put options.
+
+  #### Example
+
+  ```python
+    # Price a batch of 5 american call options.
+    volatilities = np.array([0.2, 0.2, 0.2, 0.2, 0.2])
+    forwards = np.array([80.0, 90.0, 100.0, 110.0, 120.0])
```

Comment:
no need for `np`. Lists will do. Also, could you please add `import tf_quant_finance as tff`. We are in the process of cleaning up documentation throughout

Reply:
Agreed.